modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/test/java/com/liferay/portal/search/tuning/rankings/web/internal/searcher/RankingSearchRequestContributorTest.java

```diff
@@ -81,6 +81,21 @@ public void testContributeIsSearchEngineSolrTrue() {
 			_rankingSearchRequestContributor.contribute(searchRequest));
 	}
 
+	@Test
+	public void testContributeRankingApplied() {
```

Comment:
`testContributeRankingApply`? merge `testContributeRankingApplied` and `testContributeRankingNotApplied`.

searchRequestBuilderFactory will probably make it fail because you need to recreate the mock and than reflect it again:

```
ReflectionTestUtil.setFieldValue(
			_rankingSearchRequestContributor, "searchRequestBuilderFactory",
			searchRequestBuilderFactory);
```

can you try it?